website/pages/en/index.js

```diff
@@ -180,11 +174,11 @@ module.exports = class Index extends React.Component {
         {[
           {
             content:
-              '<p>Stakeholders of the project have a regular video-chat meetup, in English. It takes place on (some) Thursdays in the afternoon, UTC time.</p>' +
-              "<p>If you're considering using or contributing to Richie, come introduce yourself and get some help!</p>",
+              '<p>Project stakeholders regularly check in through virtual meetups in English. Meetings typically take place on Thursday afternoons, UTC.</p>' +
```

Comment:
it's now tuesdays afternoon :hand_over_mouth:

Reply:
😳 Ah! Easy enough fix.